src/Surfnet/ServiceProviderDashboard/Domain/Entity/IdentityProvider.php

```diff
@@ -49,13 +49,13 @@ public function __construct($manageId, $entityId, $nameNl, $nameEn)
     {
         Assert::stringNotEmpty($manageId);
         Assert::stringNotEmpty($entityId);
-        Assert::string($nameNl);
+        Assert::nullOrString($nameNl);
         Assert::string($nameEn);
```

Comment:
Not sure if this is an actual business rule, but the nameEN is always set, so we can use stringNotEmpty on the $nameEn parameter here right?

Reply:
Thanks, double checked it in manage and that seems a valid assumption so I'll change this.